static/app/gettingStartedDocs/javascript/nextjs.spec.tsx

```diff
@@ -34,4 +34,11 @@ describe('javascript-nextjs onboarding docs', () => {
       screen.getByText(textWithMarkupMatcher(/sentry-example-page/))
     ).toBeInTheDocument();
   });
+
+  it('has metrics onboarding configuration', () => {
+    expect(docs.metricsOnboarding).toBeDefined();
+    expect(docs.metricsOnboarding?.install).toBeDefined();
+    expect(docs.metricsOnboarding?.configure).toBeDefined();
+    expect(docs.metricsOnboarding?.verify).toBeDefined();
+  });
```

Comment:
shouldn't we also check if the metrics checkbox option is not selected? basically the opposite